app/lib/submission_builder/ty2024/states/id/documents/id40.rb

```diff
@@ -25,13 +25,17 @@ def document
                   xml.DependentGrid do
                     xml.DependentFirstName sanitize_for_xml(dependent.first_name, 20)
                     xml.DependentLastName sanitize_for_xml(dependent.last_name, 20)
-                    unless dependent.ssn.nil?
+                    unless dependent.ssn.nil?  # this will produce invalid xml if it's missing
```

Comment:
This was a note to myself, I've deleted it. I was just noting that if we ever do hit this branch and omit the `DependentSSN` in the output XML, we'll get a validation error - but we shouldn't hit this branch since there should always be an ssn (TIN really) on the dependent